EleksTubeHAX/GLOBAL_DEFINES.h

```diff
@@ -32,17 +40,23 @@
 #define WIFI_CONNECT_TIMEOUT_SEC  20
 #define WIFI_RETRY_CONNECTION_SEC  15
 
-#define GEOLOCATION_API_KEY " api key "  // free for 5k loopkups per month. Get yours on https://www.abstractapi.com/ (login) --> https://app.abstractapi.com/api/ip-geolocation/tester (key)
 
-// ************ MQTT config *********************
+#define HARDWARE_NO_WPS  ////comment to use WPS 

```

Comment:
Should this be defined inside `#define HARDWARE_NovelLife_SE_CLOCK`?  Is there a time you'd want to NOT use WPS that is not associated with the hardware used?

Also: This is `NO WPS` but everywhere it's used, it's used as `#ifndef`.  The double negative is hard to follow.  I'd prefer to see this changed to `#define HARDWARE_WPS_SUPPORT` and change the conditionals to `#ifdef` instead.  That makes more sense to me, easier to follow what's going on.

Reply:
I understand -- using ifndef seemed like a smaller change to the original code.  I can reverse it.

Reply:
It's the same change, it's just optimizing out a double negative.

Reply:
I get it.  My mind isn't always linear :-)